src/java.base/share/classes/java/util/random/RandomGenerator.java

```diff
@@ -634,9 +640,12 @@ default int nextInt() {
      *
      * @throws IllegalArgumentException if {@code bound} is not positive
      *
-     * @implSpec The default implementation simply calls
-     * {@link RandomSupport#checkBound checkBound}(bound) and then
-     * {@link RandomSupport#boundedNextInt boundedNextInt}(this, bound).
+     * @implSpec The default implementation checks that {@code bound} is a
+     * positive int. Then invokes {@code nextInt()}, limiting the result to
```

Comment:
I think if you are refering to the Java type it should be in {@ code xxx }.
"int" - > {@ code int}.  Or use "integer".
Ditto "ints", "long", and "longs" in the overridden methods below.

Reply:
Updated